### ISSUE TITLE
how to update antq itself

### ISSUE THREAD

**Issue by vinurs** [liquidz/antq#146]
hello, i install antq as clojure tool
```
clojure -Ttools install com.github.liquidz/antq '{:git/tag "1.3.2"}' :as antq
```
and it generate a file called `antq.edn` in `~/.clojure/tools`, then how can i use antq to check its own version  and update to the latest

**Comment by liquidz** [comment]
@vinurs Sorry for late reply.
Currently, there is no way to update antq which is installed as clojure tool.
But it is very interesting and useful if we can do it :)

**Comment by vinurs** [comment]
yeah, i think this would be a uesful feature

**Comment by liquidz** [comment]
@vinurs
I've added `--check-clojure-tools` option in [checking-clojure-tools](https://github.com/liquidz/antq/tree/feature/checking-clojure-tools) branch.
This option allows you to detect all tools intalled in `~/.clojure/tools` as dependencies, and you can upgrade them with `--upgrade` option.

Could you try this branch?

**Comment by vinurs** [comment]
@liquidz thanks very much, i've checked the options, it works well

**Comment by liquidz** [comment]
@vinurs Thanks for your confirmation!

Currently, I'm thinking of enabling this feature by default without any options.
And `--skip` option can be used to disable like other features.

How do you think? May it be noisy?

**Comment by vinurs** [comment]
hello,in my opinion when i change dir in a project directory, i just want to check and update the project deps, but when i in a dir not belong to any project, then i want to check the global deps, eg (clojure tools, .lein/profiles.clj and so on)

**Comment by liquidz** [comment]
@vinurs Thanks!

Personally, I want this feature to be eather ON of OFF by default, because changing the behavior depending on the current directory may cause confusion.

As you said, it is resonable to check and update only project deps in our project directory, since installed Clojure CLI Tools are not related to our project.

So it may be good to disable by default.

**Comment by vinurs** [comment]
@liquidz well, agree with you

**Comment by liquidz** [comment]
@vinurs Just released v1.6.0!
